services/libs/opensearch/src/service/organization.sync.service.ts

```diff
@@ -346,6 +305,8 @@ export class OrganizationSyncService {
         }
       }
 
+      await this.indexingRepo.markEntitiesIndexed(IndexedEntityType.ORGANIZATION, organizationIds)
```

Comment:
This is probably where it will get slow over time with `indexed_entities`. Just wanted to make sure we have considered this since I never put much attention to the table with the indexes and queries... It worked before when we had ~20 mil. activities and 300k members :D

Reply:
Yep, makes sense.

We don’t have many indexes or queries on this table, so I think we're good for now. But yeah, we should definitely look into this more before it starts slowing down over time.